### PR TITLE
Update forum-qualitative-social-research.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via email for some smaller fixes.
Two outstanding questions I have:
How can I achieve the en-dash between issued dates at the end?
Hisakata, Rumi; Nishida, Shin’ya & Johnston, Alan (2016). An adaptable metric shapes perceptual space. Current Biology, 26(14), 1911–1915.

How can CSL or the user achieve the capital "A" in the subtitle?
Sambrook, Joe, & Russell, David William (2001). Molecular cloning: A laboratory manual (3rd ed.). Cold Spring Harbor, NY: CSHL Press.

### CODE REVIEW

File: forum-qualitative-social-research.csl [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
-  <!-- Polyglot; journal publishes in English, German, and (maybe?) Spanish -->
```
Comment (rmzelle) [inline forum-qualitative-social-research.csl]: @POBrien333, I assume you didn't remove this on purpose?

**Comment by POBrien333** [comment]
Sorry.  Forgot they. Will add back in.

File: forum-qualitative-social-research.csl (hunk 1 above)
Comment (POBrien333) [reply forum-qualitative-social-research.csl]: Sorry. Forgot they. Will add back in.
Any idea iwth the questions, I had, @rmzelle ?

**Comment by rmzelle** [comment]
> How can I achieve the en-dash between issued dates at the end?
> Hisakata, Rumi; Nishida, Shin’ya & Johnston, Alan (2016). An adaptable metric shapes perceptual space. Current Biology, 26(14), 1911–1915.

I think this is the default. See http://docs.citationstyles.org/en/stable/specification.html#date-ranges.

**Comment by rmzelle** [comment]
> How can CSL or the user achieve the capital "A" in the subtitle?
> Sambrook, Joe, & Russell, David William (2001). Molecular cloning: A laboratory manual (3rd ed.). Cold Spring Harbor, NY: CSHL Press.

@adam3smith?

**Comment by POBrien333** [comment]
This one should be ready to merge then.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**